apps/coscrad-frontend/src/coscrad-theme-provider.tsx

```diff
@@ -35,6 +35,14 @@ export const CoscradThemeProvider = ({ children }: CoscradThemeProviderProps): J
                     fontWeight: 'bold',
                     marginBottom: '7px',
                 },
+                h6: {
```

Comment:
I never thought we'd reach `h6`. I'll trust your judgment, but that seems pretty extreme.